examples/firewall/firewallConfig.go

```diff
@@ -0,0 +1,461 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+	"github.com/aws/aws-sdk-go/service/networkfirewall"
+	"github.com/aws/aws-sdk-go/service/networkfirewall/networkfirewalliface"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+type Client struct {
+	ec2Client      ec2iface.EC2API
+	firewallClient networkfirewalliface.NetworkFirewallAPI
+}
+
+func main() {
+	var profile string
+	var region string
+	var awsClient Client
+
+	awsProfile := flag.String("p", "", "aws profile")
+	awsRegion := flag.String("r", "", "aws region")
+	flag.Parse()
+	if *awsProfile != "" {
+		fmt.Println("Profile: ", *awsProfile)
+		profile = *awsProfile
+	} else {
+		profile = os.Getenv("AWS_PROFILE")
+	}
+	if *awsRegion != "" {
+		fmt.Println("Region: ", *awsRegion)
+		region = *awsRegion
+	} else {
+		region = os.Getenv("AWS_REGION")
+	}
+
+	if profile == "" {
```

Comment:
This is also being checked in the NewClient function. I think it can be removed from here.